src/pages/home/report/ReportActionsView.js

```diff
@@ -110,37 +126,46 @@ function ReportActionsView(props) {
             }
         });
         return () => {
-            if (!unsubscribeVisibilityListener.current) {
+            if (!unsubscribeVisibilityListener) {
                 return;
             }
-            unsubscribeVisibilityListener.current();
+            unsubscribeVisibilityListener();
         };
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, []);
+    }, [isReportFullyVisible, isFocused, props.report, reportID, setNewMarkerReportActionID]);
```

Comment:
Is the `reportID` needed if its part of `props.report`? Can you leave a comment why?
```suggestion
    }, [isReportFullyVisible, isFocused, props.report, setNewMarkerReportActionID]);
```

Reply:
Whoops, the reportID can be removed, you are right, thx!